test/image-blueprints/layer4-art/group1/rhel92_microshift-art.toml

```diff
@@ -0,0 +1,39 @@
+name = "rhel-92-microshift-art-base"
+description = "A RHEL 9.2 image with the RPMs built from ART."
+version = "0.0.1"
+modules = []
+groups = []
+distro = "rhel-92"
+
+[[packages]]
+name = "microshift"
+version = "{{ .Env.SOURCE_VERSION }}"
```

Comment:
I'm not sure we should be reusing the same `SOURCE_VERSION` setting for something else. 
May I suggest using the notion of external rpms for what you're trying to implement. 

We could have a special `EXTERNAL_SOURCE_VERSION` setting that would be valid during the build only if a dedicated directory is populated with RPMs manually.

Reply:
The build should fail if such RPMs do not exist.